src/routes/current-studies/StudyCard.svelte

```diff
@@ -1,3 +1,39 @@
+<script context=module>
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+ // ^^^ the linter currently forces this block even though it appears below.
```

Comment:
Do we have an issue about this? Can you file one if we don't?

Reply:
yep, we have one filed.